wled00/wled.h

```diff
@@ -37,6 +37,7 @@
 #ifndef WLED_DISABLE_WEBSOCKETS
   #define WLED_ENABLE_WEBSOCKETS
 #endif
+#define WLED_ENABLE_OPC
```

Comment:
You could add #ifndef WLED_DISABLE_OPC to allow complation without OPC. Similar as a line above.